docs/source/sync_in_detail.rst

```diff
@@ -0,0 +1,53 @@
+Lochness ``sync.py`` function in detail
+=======================================
+
+Lochness provides a single command line tool (daemon) to periodically poll
+and download data from various web services into a local directory. Out of
+the box there is support for pulling data from a multitude of 
+`data sources <data_sources.html>`_ including REDCap, XNAT, Dropbox, Box,
+Mediaflux, RPMS, external hard drives, and more.
+
+
+1. Loads configuration file
+---------------------------
+``Lochness`` loads and sets up configurations based on the information
+contained in the ``config.yml``.
+
+
+
+2. Automatically create and update metadata for each site (study)
+-----------------------------------------------------------------
+``Lochness`` connects to either ``REDCap`` or ``RPMS`` to download list of
+subjects registered for each site (``study``) and creates ::
+
+    PHOENIX
+    └── GENERAL
+        ├── PronetAB
+        │   └── PronetAB_metadata.csv
+        └── PronetCD
+            └── PronetCD_metadata.csv
+
+
+eg) ``PronetAB_metadata.csv``  ::
+
+    Active,Consent,Subject ID,REDCap,Box,XNAT,Mindlamp
```

Comment:
Probably no action is necessary but to import in DPdash, we must have a `Study` column.